### PR TITLE
Updated Alpha Stacker instructions

### PR DESCRIPTION
**Pull request by chortlehoort**
Exhaustive instructions for Alpha Stacker exercise since we want to use it for pre-work. Since this will be sent to students before their first day in class, I've added instructions that provide much more guidance based on what they will be doing while at NSS.

### CODE REVIEW

File: 2-the-static-web/exercises/SW_JS_ALPHASTACKER.md [hunk 1]
```diff
@@ -1,4 +1,39 @@
-Create a stackLetters function that should console.log rows of letters. It should take an array containing the letters of the alphabet and output them like this:
+# Alpha Stacker
+
+## Instructions
+
+For this exercise, you need to create a single JavaScript function named `stackLetters` that will use `console.log()` to print out letter in the alphabet to the browser console.
```
Comment (JoeShep) [inline 2-the-static-web/exercises/SW_JS_ALPHASTACKER.md]: "letter" should be "letters"

File: 2-the-static-web/exercises/SW_JS_ALPHASTACKER.md [hunk 2]
```diff
@@ -1,4 +1,39 @@
...
+
+1. If you haven't done so yet, create the `workspace/exercises/alpha-stacker` directory structure in your home folder.
```
Comment (JoeShep) [inline 2-the-static-web/exercises/SW_JS_ALPHASTACKER.md]: Not sure everyone will know what the home folder is. Your commands below don't necessarily require them to know that, I guess. Unless they don't know what the console is at this point 😱

**Comment by JoeShep** [comment]
Couple of nitpicks, but otherwise this looks good. Your guess is as good as mine as to whether unguided newbies will be able to navigate this and get it up and running. The first feedback will be interesting

**Comment by chortlehoort** [comment]
@JoeShep I added more clarity around the change directory step and fixed the misspelled word.

**Comment by chortlehoort** [comment]
👍 Per @JoeShep

**Comment by studio6thave** [comment]
The update looks good.

**Pull request merged**